queries/php/highlights.scm

```diff
@@ -108,6 +108,12 @@
 
 ; Conditions ( ? : )
 (conditional_expression) @conditional
+
+; Directives
+(declare_directive "strict_types" @preproc)
+(declare_directive "ticks" @preproc)
+(declare_directive "encoding" @preproc)
```

Comment:
```suggestion
(declare_directive ["strict_types" "ticks" "encoding"] @preproc)
```

Reply:
And are there other keywords for declare?
If there is then maybe you should do something else

Reply:
I didn't know this syntax, thanks!

Reply:
There are only these three directives in [the parser](https://github.com/tree-sitter/tree-sitter-php/blob/973694ffcdeebca245b7ecf0d7c4cadd4f41b3c9/grammar.js#L566).